python_modules/dagster/dagster_tests/model_tests/test_dagster_model.py

```diff
@@ -67,8 +67,6 @@ class MyModel(DagsterModel):
 
 
 def test_cached_method() -> None:
-    assert DagsterModel._cached_method_cache__internal__.__name__ == CACHED_METHOD_CACHE_FIELD  # noqa # type: ignore
```

Comment:
what was intended to be covered here is in practice covered by cached_method working at all as tested below